README.md

```diff
@@ -76,6 +77,35 @@ EXAMPLES
 
 _See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v0.1.0/src/commands/autocomplete/index.ts)_
 
+## `chectl devfile:generate`
+
+generate and print a devfile to stdout given some Kubernetes resources and other Che workspaces features (project, language-support, commands etc...)
+
+```
+USAGE
+  $ chectl devfile:generate
+
+OPTIONS
+  -h, --help                 show CLI help
+  --command=command          Command to include in the workspace
+  --dockerImage=dockerImage  dockerImage component to include to the workspace
+  --editor=editor            Editor to choose. Currently supported editors: theia-next,theia-1.0.0
```

Comment:
It makes sense to specify default value here.

Reply:
Actually the default is no editor (in the `Devfile`) that is interpreted by Che as "use the default editor component" (currently `theia:next`). Hence I am not sure if and what should be mentioned here as default.